packages/api/src/dataProviders/db/uniteLegalName/UniteLegalNameDbo.ts

```diff
@@ -0,0 +1,10 @@
+import { Siren } from "dto";
+import { ObjectId } from "mongodb";
+
+export default interface UniteLegalNameDbo {
+    siren: Siren;
+    name: string;
+    searchingKey: string;
```

Comment:
```suggestion
    searchKey: string;
```
J'ai l'impression d'être pénible mais c'est le moment

Reply:
done